src/sc_keeper/routers/administrative.py

```
@@ -70,3 +71,122 @@ def _count(table):
+        has_benchmarks = server["has_benchmarks"]
+
+        if is_active:
+            vendor_stats[vendor_id]["active"] += 1
+        if is_active and has_benchmarks:
+            vendor_stats[vendor_id]["evaluated"] += 1
+        elif is_active and has_price and not has_benchmarks:
+            vendor_stats[vendor_id]["missing"] += 1
+        elif not is_active or not has_price:
+            vendor_stats[vendor_id]["inactive"] += 1
+        else:
+            raise ValueError(f"Unexpected server category: {server}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)